reference/employer/openapi.yaml

```diff
@@ -523,7 +523,7 @@ paths:
                               nullable: true
                             class_code:
                               type: string
-                              description: "Worker's compensation classification code for this employee"
+                              description: Worker's compensation classification code for this employee
```

Comment:
is this expected?

Reply:
yeah stoplight editor just saves files with weird formatting